.github/workflows/reusable_publish_docs.yml

```diff
@@ -5,6 +5,13 @@ env:
 
 on:
   workflow_call:
+    secrets:
+      AWS_DOCS_ROLE_ARN:
+        description: Docs IAM Role
+        required: false
+      AWS_DOCS_BUCKET:
+        description: Docs S3 Bucket
+        required: false
```

Comment:
use `secrets: inherit` instead from the caller

Reply:
In addition to this, use `environment: Docs` in the caller, and move the secrets into the `Docs` environment please. Don't leave any "Repository secrets".